pkgs/applications/misc/remarkable/rmview/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, python3Packages, wrapQtAppsHook }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "rmview";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "bordaigorl";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0zyngirpg808k1pkyhrk43qr3i8ilvfci0wzwk4b5f6f9cmjs7kb";
+  };
+
+  nativeBuildInputs = with python3Packages; [ pyqt5 wrapQtAppsHook ];
```

```suggestion
  nativeBuildInputs = with python3Packages; [ pyqt5 wrapQtAppsHook ];
```
You are sure about pyqt5 here?

---

Yes, it's needed to have `pyrcc5` in line 18